MapboxNavigation/Resources/Base.lproj/Localizable.strings

```diff
@@ -19,6 +19,9 @@
 /* Delimiter between route numbers in a road concurrency */
 "REF_DELIMITER" = " / ";
 
+/* Delimiter between destinations in a road concurrency */
```

Comment:
A road concurrency is when a stretch of road is part of multiple routes. This delimiter is merely used when there are multiple destinations, so not necessarily a road concurrency.